### PR TITLE
Feature/update profile

### PR DESCRIPTION
**Pull request by jaimejazarenoiii**
# 📲 What #

* Update profile mutation

# 🤔 Why #

* New feature

# 🛠 How #

* Expose updateProfile mutation

# ✅ Acceptance criteria #

- [x] Test created and all is success
- [x] Rubocop success

# ⏰ TODO #

* N/A

# 🔗 References #

* https://trello.com/c/ISGDyxHs/21-update-profile-api

### CODE REVIEW

File: test/graphql/mutations/update_profile_test.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                     password_confirmation: password)
+      post('/graphql',
+           params: {
+             query: sign_in_with_email_mutation,
+             variables: sign_in_with_email_mutation_variables({ email: @user.email,
+                                                                password: password })
+           })
+
+      @json_response = parse_graphql_response(response.body)
+      @token = @json_response.dig('signInWithEmail', 'token')
```
Comment (rexendy) [inline test/graphql/mutations/update_profile_test.rb]: @jaimejazarenoiii , pls remove @ if the variable is used in function only

File: test/graphql/mutations/update_profile_test.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      post('/graphql',
+           params: {
+             query: update_profile_mutation,
+             variables: update_profile_mutation_variables({ first_name: expected_first_name })
+           }, headers: {
+             'Authorization': "Bearer #{@token}"
+           })
+
+      @json_response = parse_graphql_response(response.body)
+      assert_equal @json_response.dig('updateProfile', 'firstName'), expected_first_name
```
Comment (rexendy) [inline test/graphql/mutations/update_profile_test.rb]: same here

File: test/graphql/mutations/update_profile_test.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+           params: {
+             query: update_profile_mutation,
+             variables: update_profile_mutation_variables({ age: age_input })
+           }, headers: {
+             'Authorization': "Bearer #{@token}"
+           })
+
+      @json_response = parse_graphql_errors(response.body)
+
+      assert_equal expected_err_mssg, @json_response[0]['message']
```
Comment (rexendy) [inline test/graphql/mutations/update_profile_test.rb]: same here and remove extra next line to be same in other test case methods

**Review by rexendy — CHANGES_REQUESTED** [review]

File: test/graphql/mutations/update_profile_test.rb (hunk 1 above)
Comment (jaimejazarenoiii) [reply test/graphql/mutations/update_profile_test.rb]: nice catch 👏🏻 thanks

**Review by rexendy — APPROVED** [review]

**Pull request merged**